apps/web/src/components/Navbar/ProfileMenu.tsx

```diff
@@ -71,6 +72,7 @@ interface LinkDetail {
   label: string
   icon: string
   href?: string
+  newTab?: boolean
```

Comment:
nit, but maybe more ergonomic
```suggestion
  openInNewTab?: boolean
```